CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to @ftrack/api
+
+Thank you for wanting to contribute to our Javascript API!
+
+Please clone this repository, and follow these instructions to make sure you are in a clean spot before coding:
+
+Using node version as reflected in `.nvmrc`, run:
+
+- yarn install
+- yarn test
+
+If the tests passes, you should be good to go. Please submit a PR with your changes and we will look at them as soon as possible.
+
+## Publishing a release
+
+_This is only relevant for ftrack employees when preparing and publishing a new release._
+
+To publish a new release, follow these steps:
+
+Locally:
+
+- `git switch -c release/<new-release-tag>`
+- `yarn version <new-release-tag>`
```

Comment:
Are we using yarn v1 or v2 here? This looks like yarn version from v1. I don't think the [v2 command](https://yarnpkg.com/cli/version) works like this.

Reply:
I'll change to `npm version <new-release-tag>` instead.

Reply:
I don't like mixing yarn and npm, but there doesn't seem to be good way of achieving the previous behavior without it.

What yarn folks seem to recommend is that you should run e.g. `yarn version minor --deferred` when doing a minor change, and then as part of the release you would then apply it using `yarn version apply`. If we want to go with this approach, we would need to enforce that all PRs include a version change.

For @ftrackhq/frontend I worked around this by using yarn version --immediate, and then remove the artifacts it created. Not ideal either.

Reply:
we decided to use `npm version` for now